css-container-queries.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
```

Comment:
If we just add a `<title>` the document becomes 100% valid HTML:

```suggestion
<!DOCTYPE html>
<title>CSS container queries</title>
```

Reply:
Thanks, done!